pkg/runtime/runtime-manifest.go

```diff
@@ -518,3 +518,13 @@ func resolveImage(image *manifest.MappedImage, refString string) error {
 	}
 	return nil
 }
+
+// toLowerCase returns a string with the first character lowercased
+func toLowerCase(str string) string {
```

Comment:
The name of this function is misleading. Perhaps it should be called `toCamelCase`?

https://medium.com/better-programming/string-case-styles-camel-pascal-snake-and-kebab-case-981407998841

Reply:
👍 `toCamelCase` it shall be.